src/Symfony/Component/Serializer/Normalizer/BackedEnumNormalizer.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+
+/**
+ * Normalizes a {@see \BackedEnum} enumeration to a string or an integer.
+ *
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+class BackedEnumNormalizer implements NormalizerInterface, DenormalizerInterface, CacheableSupportsMethodInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return string
+     */
+    public function normalize($object, string $format = null, array $context = [])
+    {
+        if (\PHP_VERSION_ID < 80100) {
```

Comment:
Well, It looks like this class is totally valid with PHP < 8.
One could create a polyfill `class \BackedEnum { public $value; public static function from(); }` and use this  normaliser.

If we really want to disable the normalized, it'd rather do it in `FrameworkBundle/Resources/config/serializer.php`
By the way the normalizer should be declared in the file ;)

Reply:
Got it!
I'll update my `serializer.php`. If I understand it right, I would result in something like this?

```php
if (\PHP_VERSION_ID < 80100) {
    $container
        ->services()
        ->set('serializer.normalizer.backed_enum', BackedEnumNormalizer::class)
        ->tag('serializer.normalizer', ['priority' => -915]);
}
```

Reply:
I would enable it depending on the existence of \BackedEnum because this is what counts. A polyfill would introduce this class and works fine 🙌

Reply:
I agree with this idea, I didn't think of polyfills. Added this snippet at the bottom of `serializer.php`:

```php
if (class_exists('\BackedEnum') || interface_exists('\BackedEnum') || trait_exists('\BackedEnum')) {
    $container->services()
        ->set('serializer.normalizer.backed_enum', BackedEnumNormalizer::class)
            ->tag('serializer.normalizer', ['priority' => -915])
    ;
}
```